kubernetes-examples/src/main/resources/test-csr-v1.yml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: certificates.k8s.io/v1
+kind: CertificateSigningRequest
+metadata:
+  name: my-cert
+spec:
+  groups:
+    - system:authenticated
+  request: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURSBSRVFVRVNULS0tLS0KTUlJQ1ZqQ0NBVDRDQVFBd0VURVBNQTBHQTFVRUF3d0dZVzVuWld4aE1JSUJJakFOQmdrcWhraUc5dzBCQVFFRgpBQU9DQVE4QU1JSUJDZ0tDQVFFQTByczhJTHRHdTYxakx2dHhWTTJSVlRWMDNHWlJTWWw0dWluVWo4RElaWjBOCnR2MUZtRVFSd3VoaUZsOFEzcWl0Qm0wMUFSMkNJVXBGd2ZzSjZ4MXF3ckJzVkhZbGlBNVhwRVpZM3ExcGswSDQKM3Z3aGJlK1o2MVNrVHF5SVBYUUwrTWM5T1Nsbm0xb0R2N0NtSkZNMUlMRVI3QTVGZnZKOEdFRjJ6dHBoaUlFMwpub1dtdHNZb3JuT2wzc2lHQ2ZGZzR4Zmd4eW8ybmlneFNVekl1bXNnVm9PM2ttT0x1RVF6cXpkakJ3TFJXbWlECklmMXBMWnoyalVnald4UkhCM1gyWnVVV1d1T09PZnpXM01LaE8ybHEvZi9DdS8wYk83c0x0MCt3U2ZMSU91TFcKcW90blZtRmxMMytqTy82WDNDKzBERHk5aUtwbXJjVDBnWGZLemE1dHJRSURBUUFCb0FBd0RRWUpLb1pJaHZjTgpBUUVMQlFBRGdnRUJBR05WdmVIOGR4ZzNvK21VeVRkbmFjVmQ1N24zSkExdnZEU1JWREkyQTZ1eXN3ZFp1L1BVCkkwZXpZWFV0RVNnSk1IRmQycVVNMjNuNVJsSXJ3R0xuUXFISUh5VStWWHhsdnZsRnpNOVpEWllSTmU3QlJvYXgKQVlEdUI5STZXT3FYbkFvczFqRmxNUG5NbFpqdU5kSGxpT1BjTU1oNndLaTZzZFhpVStHYTJ2RUVLY01jSVUyRgpvU2djUWdMYTk0aEpacGk3ZnNMdm1OQUxoT045UHdNMGM1dVJVejV4T0dGMUtCbWRSeEgvbUNOS2JKYjFRQm1HCkkwYitEUEdaTktXTU0xMzhIQXdoV0tkNjVoVHdYOWl4V3ZHMkh4TG1WQzg0L1BHT0tWQW9FNkpsYWFHdTlQVmkKdjlOSjVaZlZrcXdCd0hKbzZXdk9xVlA3SVFjZmg3d0drWm89Ci0tLS0tRU5EIENFUlRJRklDQVRFIFJFUVVFU1QtLS0tLQo=
```

Comment:
Certificate Request:
    Data:
        Version: 0 (0x0)
        Subject:
            commonName                = angela
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (2048 bit)
                Modulus:
                    00:d2:bb:3c:20:bb:46:bb:ad:63:2e:fb:71:54:cd:
                    91:55:35:74:dc:66:51:49:89:78:ba:29:d4:8f:c0:
                    c8:65:9d:0d:b6:fd:45:98:44:11:c2:e8:62:16:5f:
                    10:de:a8:ad:06:6d:35:01:1d:82:21:4a:45:c1:fb:
                    09:eb:1d:6a:c2:b0:6c:54:76:25:88:0e:57:a4:46:
                    58:de:ad:69:93:41:f8:de:fc:21:6d:ef:99:eb:54:
                    a4:4e:ac:88:3d:74:0b:f8:c7:3d:39:29:67:9b:5a:
                    03:bf:b0:a6:24:53:35:20:b1:11:ec:0e:45:7e:f2:
                    7c:18:41:76:ce:da:61:88:81:37:9e:85:a6:b6:c6:
                    28:ae:73:a5:de:c8:86:09:f1:60:e3:17:e0:c7:2a:
                    36:9e:28:31:49:4c:c8:ba:6b:20:56:83:b7:92:63:
                    8b:b8:44:33:ab:37:63:07:02:d1:5a:68:83:21:fd:
                    69:2d:9c:f6:8d:48:23:5b:14:47:07:75:f6:66:e5:
                    16:5a:e3:8e:39:fc:d6:dc:c2:a1:3b:69:6a:fd:ff:
                    c2:bb:fd:1b:3b:bb:0b:b7:4f:b0:49:f2:c8:3a:e2:
                    d6:aa:8b:67:56:61:65:2f:7f:a3:3b:fe:97:dc:2f:
                    b4:0c:3c:bd:88:aa:66:ad:c4:f4:81:77:ca:cd:ae:
                    6d:ad
                Exponent: 65537 (0x10001)
        Attributes:
            a0:00
    Signature Algorithm: sha256WithRSAEncryption
         63:55:bd:e1:fc:77:18:37:a3:e9:94:c9:37:67:69:c5:5d:e7:
         b9:f7:24:0d:6f:bc:34:91:54:32:36:03:ab:b2:b3:07:59:bb:
         f3:d4:23:47:b3:61:75:2d:11:28:09:30:71:5d:da:a5:0c:db:
         79:f9:46:52:2b:c0:62:e7:42:a1:c8:1f:25:3e:55:7c:65:be:
         f9:45:cc:cf:59:0d:96:11:35:ee:c1:46:86:b1:01:80:ee:07:
         d2:3a:58:ea:97:9c:0a:2c:d6:31:65:30:f9:cc:95:98:ee:35:
         d1:e5:88:e3:dc:30:c8:7a:c0:a8:ba:b1:d5:e2:53:e1:9a:da:
         f1:04:29:c3:1c:21:4d:85:a1:28:1c:42:02:da:f7:88:49:66:
         98:bb:7e:c2:ef:98:d0:0b:84:e3:7d:3f:03:34:73:9b:91:53:
         3e:71:38:61:75:28:19:9d:47:11:ff:98:23:4a:6c:96:f5:40:
         19:86:23:46:fe:0c:f1:99:34:a5:8c:33:5d:fc:1c:0c:21:58:
         a7:7a:e6:14:f0:5f:d8:b1:5a:f1:b6:1f:12:e6:54:2f:38:fc:
         f1:8e:29:50:28:13:a2:65:69:a1:ae:f4:f5:62:bf:d3:49:e5:
         97:d5:92:ac:01:c0:72:68:e9:6b:ce:a9:53:fb:21:07:1f:87:
         bc:06:91:9a